extensions/amp-nexxtv-player/0.1/amp-nexxtv-player.js

```diff
@@ -172,27 +170,24 @@ class AmpNexxtvPlayer extends AMP.BaseElement {
   sendCommand_(command) {
     this.playerReadyPromise_.then(() => {
       if (this.iframe_ && this.iframe_.contentWindow) {
-        this.iframe_.contentWindow./*OK*/postMessage(JSON.stringify({
-          'cmd': command,
-        }), '*');
+        this.iframe_.contentWindow./*OK*/postMessage({cmd: command}, '*');
       }
     });
   };
 
   // emitter
   handleNexxMessages_(event) {
-    const data = isObject(event.data) ? event.data : tryParseJson(event.data);
-    if (data === undefined) {
-      return; // We only process valid JSON.
+    if (event.data === undefined) {
```

Comment:
Since we are listening to `message` event on the parent window, these events can come from any iframe, not just nexxtv-player, so we need to account for that with something like:
```
    if (!event.data ||
        event.source != this.iframe_.contentWindow) {
      return;
    }
```